README.md

```
@@ -13,3 +16,71 @@
+### ⏰ 데일리 스크럼 & 회고
+- 🕙 **데일리 스크럼**: 오전 `10:00 ~ 10:30`  
+  - 진행 상황 공유 & 오늘의 작업 계획 수립  
+- 🕡 **데일리 회고**: 퇴근 전 `18:30 ~ 19:00`  
+  - 하루 동안의 업무 피드백 및 팀원 간의 소통
+
+---
+
+# 🤔 고민했던 사항
+> **저희 팀이 마주했던 고민은 GitHub Wiki에서 확인할 수 있습니다!**  
+🔗 [백엔드 Wiki](https://github.com/softeer5th/team1-dingdong/wiki/Backend)  
+🔗 [프론트엔드 Wiki]()
```

markdownlint-cli2 (0.17.2)

No empty links
null

(MD042, no-empty-links)